### PR TITLE
Improve error handling when user tries to create file under root (user resource)

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
@jeffbaumes Does this message look better? Feel free to suggest something better.

File: girder_jupyter/contents/manager.py [hunk 1]
```diff
@@ -410,8 +410,9 @@ def _upload_to_path(self, content, mime_type, format, path):
         parent = self._create_folders('/'.join(folder_path))
 
         if self._is_user(parent):
-            raise web.HTTPError(400, 'Can\'t create file under user.',
-                                reason='Can\'t create file under user.')
+            msg = 'Can\'t create file as a child of a Girder user. Navigate to folder ' \
+                'before creating file.'
```
Comment (jeffbaumes) [inline girder_jupyter/contents/manager.py]: I'd suggest a bit more user-friendly full sentences, something along the lines of:

"The Girder user's home location may only contain folders. Create or navigate to another folder before creating or uploading a file."

**Review by jeffbaumes — CHANGES_REQUESTED** [review]
This change works as expected for upload and creating a notebook.

I have a small suggestion for improving the error text.

**Comment by cjh1** [comment]
@jeffbaumes Thanks, much friendlier :+1:

**Review by jeffbaumes — APPROVED** [review]

**Pull request merged**